src/screens/AuthScreen/AuthScreen.tsx

```diff
@@ -37,77 +24,15 @@ const AuthScreen = () => {
   //TODO: add to constants
   const maxLength = 4;
```

Comment:
we already have a constant file, please move it to constant

Reply:
This wasn't changed in the current PR but in another one where OTPModal was added. It is going to be made irrelevant by coming changes of QR Scanner. This PR concerns only with the installation, config and fixes of react-native-vision camera.